packages/atomic/src/components/facets-v1/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -129,6 +134,25 @@ export class AtomicNumericFacet
     };
   }
 
+  @Listen('atomic/numberFormat')
+  public setFormat(event: CustomEvent<NumberFormatter>) {
+    event.preventDefault();
+    event.stopPropagation();
```

Comment:
I don't understand the purpose of these two lines.

Reply:
**preventDefault**: If invoked when the cancelable attribute value is true, and while executing a listener for the event with passive set to false, signals to the operation that caused event to be dispatched that it needs to be canceled.

It allows us to know when the event has been caught by an handler, this way, if it's not caught, we can display an error saying "The component was not handled as it is not a child of a compatible component" 

**stopPropagation**: When dispatched in a tree, invoking this method prevents event from reaching any objects other than the current object.

Basically prevents the event from bubbling up once it's caught, a safety mesure